index.html

```diff
@@ -1,21 +1,112 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Karma</title>
+    <title>Karma Abbas</title>
     <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
     <!-- Add a link to your CSS file here (use the line above to guide you) -->
     <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
 </head>
+
 <body>
+    <!-- navigation bar -->
+    <header>
+        <nav class="navbar">
+            <img class="logo" src="img/karma-logo.svg" alt="Karma Logo">
+            <ul class="links">
+                <li>
+                    <a href="#"><span class="dark">Meet Karma</span></a>
+                </li>
+                <li>
+                    <a href="#">How it Works</a>
+                </li>
+                <li>
+                    <a href="#">Store</a>
+                </li>
+                <li>
+                    <a href="#">Blog</a>
+                </li>
+                <li>
+                    <a href="#">Help</a>
+                </li>
+                <li>
+                    <a href="#">Login</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+
+    <!-- SHOWCASE -->
+    <section class="showcase">
+        <h2>
+            Introducing Karma
+        </h2>
+        <h4>
+            Bring WiFi with you, everywhere you go.
+        </h4>
+
+        <button class="button" type="button">
+            Learn More
+        </button>
 
+    </section>
+
+    <!-- SEGMENTS -->
+    <section class="seg">
+        <h2>
+            Everyone needs a little Karma.
+        </h2>
+        <div class="segments">
+            <div class="devices">
+                <img src="img/icon-devices.svg" alt="Devices" class="dev">
+                <h5>
+                    Internet for all devices
+                </h5>
+            </div>
+            <div class="cup">
+                <img src="img/icon-coffee.svg" alt="Coffee" class="coffee">
+                <h5>
+                    Boost your productivity
+                </h5>
+            </div>
+            <div class="fuel">
+                <img src="img/icon-refill.svg" alt="refill" class="nuzzle">
+                <h5>
+                    Pay as You Go
+                </h5>
+            </div>
+        </div>
+    </section>
+
+    <!-- FOOTER -->
+    <footer id="footer">
```

Comment:
Why are you using an ID here? 

Don't use IDs in your CSS!
https://stackoverflow.com/questions/8279132/why-shouldnt-i-use-id-selectors-in-css